compliance/config.py

```diff
@@ -14,6 +14,8 @@
 BUCKET_NAME = "active-storage-compliance-test-data"
 # Address of active storage proxy to be tested - setting value to 'None' will result in proxy responses being mocked by pytest
 PROXY_URL = "http://localhost:8080"
+# Optional path to CA cert to use for https requests
+PROXY_TLS_CERT = None # '/path/to/certificate.crt'
```

Comment:
nit: CA_CERT or CACERT would be more correct. CERT might be confused with a client certificate.